poetry/action.yaml

```diff
@@ -32,20 +32,20 @@ branding:
 runs:
   using: "composite"
   steps:
-    - name: Set up poetry
-      uses: actions/setup-python@v5
-      id: python
-      with:
-        python-version: ${{ inputs.python-version }}
-        cache: ${{ inputs.cache == 'true' && 'poetry' || '' }}
-        cache-dependency-path: ${{ inputs.cache-dependency-path }}
     - name: Set up poetry
       uses: greenbone/actions/pipx@v3
       with:
         python-path: ${{ steps.python.outputs.python-path }}
```

Comment:
Any idea what to do about this? We can't take the output of a later step as an input 🤔

Reply:
I wonder if it is even required, since pipx is also setting up python anyway: https://github.com/greenbone/actions/blob/main/pipx/action.yml#L41
You may just adjust it there and use the output of the step in the action itself.